pinion_adverts.sp

```diff
@@ -577,40 +607,57 @@ public OnPluginStart()
 #endif
 }
 
+public Action:FuncChatHook(Handle:event, const String:name[], bool:dontBroadcast)
+{
+	new client = GetClientOfUserId(GetEventInt(event, "userid"));
+	decl String:strChat[256]; 
+	GetEventString(event, "text", strChat, sizeof(strChat));
+
+	if (StrContains(strChat, "!RewardMe", false) == 0)
```

Comment:
yes - and thats wanted?

*from mobile*
Daniel

Am 15.04.2016 um 18:28 schrieb Caelan <notifications@github.com>:

In pinion_adverts.sp
<https://github.com/PinionTech/Pinion-Adverts/pull/30#discussion_r59901651>:

> @@ -577,40 +607,57 @@ public OnPluginStart()
>  #endif
>  }
>
> +public Action:FuncChatHook(Handle:event, const String:name[], bool:dontBroadcast)
> +{
> +	new client = GetClientOfUserId(GetEventInt(event, "userid"));
> +	decl String:strChat[256];
> +	GetEventString(event, "text", strChat, sizeof(strChat));
> +
> +	if (StrContains(strChat, "!RewardMe", false) == 0)

@decrypted <https://github.com/decrypted>
if (StrContains(strChat, "!RewardMe", false) == 0)
Is a case insensitive search.

—
You are receiving this because you were mentioned.
Reply to this email directly or view it on GitHub
<https://github.com/PinionTech/Pinion-Adverts/pull/30/files/3c434f522082ae2dc61996e040ef8297b2bbd5ea#r59901651>